model/src/w3profsmd_pdlib.F90

```diff
@@ -5585,1055 +5529,648 @@ SUBROUTINE PDLIB_JACOBI_GAUSS_SEIDEL_BLOCK(IMOD, FACX, FACY, DTG, VGX, VGY)
-    WRITE(740+IAPROC,*) 'NX=', NX
-    WRITE(740+IAPROC,*) 'NP=', NP
-    WRITE(740+IAPROC,*) 'NPA=', NPA
-    WRITE(740+IAPROC,*) 'NSEA=', NSEA
-    WRITE(740+IAPROC,*) 'NSEAL=', NSEAL
-    WRITE(740+IAPROC,*) 'NBI=', NBI
-    WRITE(740+IAPROC,*) 'B_JGS_TERMINATE_NORM=', B_JGS_TERMINATE_NORM
-    WRITE(740+IAPROC,*) 'B_JGS_TERMINATE_DIFFERENCE=', B_JGS_TERMINATE_DIFFERENCE
-    WRITE(740+IAPROC,*) 'B_JGS_TERMINATE_MAXITER=', B_JGS_TERMINATE_MAXITER
-    WRITE(740+IAPROC,*) 'B_JGS_MAXITER=', B_JGS_MAXITER
-    WRITE(740+IAPROC,*) 'B_JGS_BLOCK_GAUSS_SEIDEL=', B_JGS_BLOCK_GAUSS_SEIDEL
-    WRITE(740+IAPROC,*) 'FSREFRACTION=', FSREFRACTION
-    WRITE(740+IAPROC,*) 'FSFREQSHIFT=', FSFREQSHIFT
-    WRITE(740+IAPROC,*) 'B_JGS_LIMITER=', B_JGS_LIMITER
-    WRITE(740+IAPROC,*) 'B_JGS_BLOCK_GAUSS_SEIDEL=', B_JGS_BLOCK_GAUSS_SEIDEL
-    FLUSH(740+IAPROC)
-#endif
-#ifdef W3_DEBUGSRC
-    WRITE(740+IAPROC,*) 'optionCall=', optionCall
-    FLUSH(740+IAPROC)
-#endif
-    call print_memcheck(memunit, 'memcheck_____:'//' WW3_PROP SECTION 1')
-    !
-    ! 2.  Convert to Wave Action ---------------- *
-    !
-#ifdef W3_DEBUGSRC
-    WRITE(740+IAPROC,*) 'NSEAL =', NSEAL, 'NP    =', NP, 'NPA   =', NPA
-#endif
-#ifdef W3_DEBUGSOLVERCOH
-    CALL ALL_VA_INTEGRAL_PRINT(IMOD, "VA(np) before transform", 0)
-    CALL ALL_VA_INTEGRAL_PRINT(IMOD, "VA(npa) before transform", 1)
-#endif
-    ! We have NSEAL = NPA so the whole field is assigned
-    DO JSEA=1,NSEAL
-      IP      = JSEA
-      IP_glob = iplg(IP)
-      ISEA    = MAPFS(1,IP_glob)
-      DO ISP=1,NSPEC
-        ITH    = 1 + MOD(ISP-1,NTH)
-        IK     = 1 + (ISP-1)/NTH
-#ifdef NOCGTABLE
-        CALL WAVNU_LOCAL(SIG(IK),DW(ISEA),WN1(IK),CG1(IK))
-#else
-        CG1(IK)    = CG(IK,ISEA)
-#endif
-        VA(ISP,JSEA) = VA(ISP,JSEA) / CG1(IK) * CLATS(ISEA)
-      END DO
-    END DO
-    VAOLD = VA(1:NSPEC,1:NSEAL)
-
-#ifdef W3_DEBUGSRC
-    DO JSEA=1,NSEAL
-      WRITE(740+IAPROC,*) 'JSEA=', JSEA
-      WRITE(740+IAPROC,*) 'min/max/sum(VA)=', minval(VA(:,JSEA)), maxval(VA(:,JSEA)), sum(VA(:,JSEA))
-    END DO
-#endif
-#ifdef W3_DEBUGSOLVERCOH
-    CALL ALL_VA_INTEGRAL_PRINT(IMOD, "VA(np) just defined", 0)
-    CALL ALL_VA_INTEGRAL_PRINT(IMOD, "VA(npa) just defined", 1)
-#endif
-#ifdef W3_DEBUGSOLVER
-    FLUSH(740+IAPROC)
-    WRITE(740+IAPROC,*) 'JACOBI_SOLVER, step 4'
-    WRITE(740+IAPROC,*) 'FSSOURCE=', FSSOURCE
-    WRITE(740+IAPROC,*) 'FSREFRACTION=', FSREFRACTION
-    WRITE(740+IAPROC,*) 'FSFREQSHIFT=', FSFREQSHIFT
-    WRITE(740+IAPROC,*) 'FSGEOADVECT=', FSGEOADVECT
-    WRITE(740+IAPROC,*) 'DTG=', DTG
-#endif
-    !
-    !    init matrix and right hand side
-    !
-    call print_memcheck(memunit, 'memcheck_____:'//' WW3_PROP SECTION 2')
-    !
-    IF (.not. LSLOC) THEN
-      IF (IMEM == 1) THEN
-        ASPAR_JAC = ZERO
-      ELSE IF (IMEM == 2) THEN
-        ASPAR_DIAG_ALL = ZERO
-      ENDIF
-      B_JAC = ZERO
-    ENDIF
-    call print_memcheck(memunit, 'memcheck_____:'//' WW3_PROP SECTION 3')
```

Comment:
@aronroland there appears to be a handful more of these removals in this file.  Can you please add them back in, or mention if there is reason to remove them?

Reply:
same